### PR TITLE
Newcomers_Guide.rst: ``work load`` -> ``workload``

### PR DESCRIPTION
**Pull request by amrandahvur**
Replaces ``work load`` with ``workload``

Closes https://github.com/coala/documentation/issues/184

### CODE REVIEW

**Comment by manu-chroma** [comment]
@amrandahvur IMO you can remove this from the commit body

Replaces ``work load`` with ``workload``

Seems redundant. Follow this procedure to change the commit message,

To change your message you just use git commit --amend and then git push --force the new commit to replace the old one.

**Comment by amrandahvur** [comment]
@manu-chroma Sure, Working on it

**Comment by adtac** [comment]
Hey,

we use some bug prediction in coala which relies on the `Fixes` keyword in commit messages. To get good results from that we need to use `Closes` for normal issues instead of `Fixes` which should only be used for real bugs. (See also [the commit message docs](http://coala.io/commit).)

To change your message you just use `git commit --amend` and then `git push --force` the new commit to replace the old one.

In other words, IMO `Fixes` is better for this commit.

If you're just looking to fix an issue very quickly and not interested in contributing to coala long term, we can fix up the message for you - just tell us :).

Cheers!

**Comment by amrandahvur** [comment]
@adtac Done, Please check ?

**Comment by adtac** [comment]
ack https://github.com/coala/documentation/pull/192/commits/eb10b3da55e586d53adc91872f1abe2416255ccd

**Comment by adtac** [comment]
@rultor merge
